src/server/custom-error.model.ts

```diff
@@ -0,0 +1,39 @@
+export enum CustomErrors {
+  UserAlreadyExists = 'UserAlreadyExists',
+  InternalServerError = 'InternalServerError',
+  UnknownError = 'UnknownError',
```

Comment:
What's the difference between an UnknownError and InternalServerError?

Reply:
again thats from my code lol ^__^, agreed tho just have `InternalServerError` and remove UnknownError